core/src/main/java/ch/cyberduck/core/Local.java

```diff
@@ -444,6 +479,27 @@ public OutputStream getOutputStream(final boolean append) throws AccessDeniedExc
         return this.getOutputStream(path, append);
     }
 
+    protected FileChannel getRawChannelOutputStream(final boolean append) throws LocalAccessDeniedException {
+        try {
+            final Set<OpenOption> options = new HashSet<>();
```

Comment:
Can we extract a method to open the channel to remove the duplicated code.

Reply:
fixed